business/src/main/java/org/mskcc/cbio/portal/service/ApiService.java

```diff
@@ -74,6 +80,165 @@
 		return cancerTypeMapper.getCancerTypes(cancer_type_ids);
 	}
 
+        @Transactional
+	public List<DBAltCount> getMutations(String type, Boolean per_study, List<String> ids,  List<String> genes, List<Integer> starts, List<Integer> ends, List<String> echo) {
```

Comment:
Can we put the parameters in hashmap instead of list them all so that we can incorporate any other parameters fro the user easily?